src/fiber.cr

```diff
@@ -60,6 +60,9 @@ class Fiber
   # :nodoc:
   property previous : Fiber?
 
+  # :nodoc:
+  setter callback : Proc(Nil)?
```

Comment:
Could you please add a comment to explain the purpose of this?
Comments are welcome in other places as well ;)